src/handlebars/index.hbs

```diff
@@ -0,0 +1,14 @@
+<!-- handlebars header -->
+{{> header }}
+
+	<div id="container">
+		
+		<section id="main">
+		
```

Comment:
I was thinking we’d leave that part for mobbing?

On Wed, Jul 29, 2015 at 1:40 PM, Ben Smithgall <notifications@github.com>
wrote:

>> @@ -0,0 +1,14 @@
>> +<!-- handlebars header -->
>> +{{> header }}
>> +
>> +	<div id="container">
>> +		
>> +		<section id="main">
>> +		
> Shouldn't this be `{{ content }}` or something?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/codeforamerica/wand/pull/27/files#r35808966